lib/apiservers/portlayer/restapi/handlers/interaction_handlers.go

```diff
@@ -436,11 +438,13 @@ func (d *FlushingReader) WriteTo(w io.Writer) (written int64, err error) {
 
 	nr, er := d.readDetectInit(buf)
 	for {
+		log.Debugf("[%p] nr=%d", d, nr)
```

Comment:
Do we really need these debug logs?  Logs is already busy.  This will make it harder for us to review customer issues.